internal/util/maps.go

```diff
@@ -35,3 +40,35 @@ func CompareMaps(existing, new map[string]string) bool {
 	}
 	return CompareStructNumTypeAgnostic(existing, new)
```

Comment:
@chandankumar4 do we need to call `CompareStructNumTypeAgnostic()` since we're passing it 2 `map[string]string`?